purchases/src/test/java/com/revenuecat/purchases/PostingTransactionsTests.kt

```diff
@@ -153,13 +155,12 @@ class PostingTransactionsTests {
         clearMocks(customerInfoHelperMock)
     }
 
+    // TODO BC5: Add tests to check trial and intro durations are sent if needed.
```

Comment:
This test probably will be testing for the pricing phases. Still need to work on it.